frontend/components/service/index.js

```diff
@@ -3,182 +3,183 @@ import ServiceCardRow from './serviceCardRow';
 import Button from '@mui/material/Button';
 import styled from '../../styles/service.module.css';
 import Select from '@mui/material/Select';
-import { useEffect, useState } from 'react';
+import {useEffect, useState} from 'react';
 import MenuItem from '@mui/material/MenuItem';
 import SearchInput from './search';
 import Box from '@mui/material/Box';
 import Link from 'next/link';
 import ComboForm from './comboForm';
+import _orderBy from "lodash/orderBy";
+
 
 const ServiceComponent = (props) => {
-  const { serviceListData, toggleBlocked, deleteService } = props;
-  const [serviceListDisplay, setServiceListDisplay] = useState(serviceListData);
-  // const [searchResult, setSearchResult] = useState([]);
```

Comment:
remove this comment

Reply:
this comment is already removed, it is just shown that line 14 to line 16 is removed, it is replaced but line with + symbol.